.github/workflows/fast_build.yml

```diff
@@ -0,0 +1,49 @@
+name: Fast build
+
+on: workflow_call
```

Comment:
Maybe I'm missing something but where this workflow is called?

Reply:
You're right. I moved that code to main workflow.